android-components/components/feature/prompts/src/main/res/values/strings.xml

```diff
@@ -20,7 +20,7 @@
     <!-- Text for password field in an authentication dialog. -->
     <string name="mozac_feature_prompt_password_hint">Password</string>
     <!-- Negative confirmation that we should not save the new or updated login -->
-    <string name="mozac_feature_prompt_dont_save">Don’t save</string>
+    <string name="mozac_feature_prompt_dont_save" moz:removedIn="124" tools:ignore="UnusedResources">Don’t save</string>
```

Comment:
The cut will happen once we come back from the long weekend. So, these should be bumped to 125 since I don't think we will get to landing this before next week.

Reply:
Can we not uplift these?